Firebase/InstanceID/FIRInstanceID.m

```diff
@@ -946,9 +946,9 @@ - (void)defaultTokenWithRetry:(BOOL)retry handler:(nullable FIRInstanceIDTokenHa
                                           object:[self.defaultFCMToken copy]];
         [[NSNotificationQueue defaultQueue] enqueueNotification:tokenRefreshNotification
                                                    postingStyle:NSPostASAP];
-
-        [self performDefaultTokenHandlerWithToken:token error:nil];
       }
+
+      [self performDefaultTokenHandlerWithToken:token error:nil];
```

Comment:
aha I just realize this is a different issue that handler just simply doesn't get called. Thanks for fixing it!

Reply:
No problem!